Dockerfile

```diff
@@ -18,59 +18,104 @@ RUN apk --no-cache add \
     && rm -rf /var/cache/apk/*
 
 # Setup repo
-COPY . ${GOPATH}/src/code.gitea.io/gitea
 WORKDIR ${GOPATH}/src/code.gitea.io/gitea
 
+COPY ./go.mod .
```

Comment:
Also go.sum?

Reply:
Have updated to include `go.sum` as well as making use of the Makefile targets `deps-backend` + `deps-frontend`